equinix/resource_network_device_link.go

```diff
@@ -128,7 +151,15 @@ func createNetworkDeviceLinkResourceSchema() map[string]*schema.Schema {
 			Type:         schema.TypeString,
 			Optional:     true,
 			ValidateFunc: validation.IsCIDR,
-			Description:  networkDeviceLinkSchemaNames["Subnet"],
+			Description:  networkDeviceLinkDescriptions["Subnet"],
+		},
+		networkDeviceLinkSchemaNames["RedundancyType"]: {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			Default:      "Primary",
+			ValidateFunc: validation.StringInSlice([]string{"Primary", "Secondary", "Hybrid"}, true),
```

Comment:
You can mark ignoreCase as false because all the available inputs will be shown in the terraform.